SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -114,7 +114,7 @@
   <!-- **************************************************************************** -->
   <!-- **************************************************************************** -->
 
-  <Target Name="SkippingSonarQubeAnalysis" BeforeTargets="CoreCompile"
+  <Target Name="SkippingSonarQubeAnalysis" BeforeTargets="Build"
           Condition=" $(SonarQubeTempPath) == '' ">
```

Comment:
Please add this remark (that Build covers both managed and C++) to the comments section for this target